### PR TITLE
Update typechain-polkadot-follow-up.md

### PR DESCRIPTION
**Pull request by coreggon11**
#### Project Abstract

This is an update of an already [accepted grant](https://github.com/w3f/Grants-Program/blob/master/applications/typechain-polkadot-follow-up.md).

We have made some team and wallet adjustments, however, we have been updating this application once, and I have noticed that after [this PR](https://github.com/w3f/Grants-Program/commit/57ef638f2bb331c24c569bff02f04c2b3e89bfb0) the changes we previously made to the grant application and which were accepted in [this PR](https://github.com/w3f/Grants-Program/pull/1236) were discarded, so I am adding these changes too.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $30,000, 3 approvals
- [X] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [X] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [X] A BTC, Ethereum (USDC/DAI) or Polkadot/Kusama (aUSD/USDT) address for the payment of the milestones is provided in the application.
- [X] The software delivered for this grant will be released under an open-source license specified in the application.
- [X] The initial PR contains only one commit (squash and force-push if needed).
- [X] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).

### CODE REVIEW

**Review by Noc2 — DISMISSED** [review]

**Review by keeganquigley — DISMISSED** [review]

**Review by alxs — APPROVED** [review]

**Review by Noc2 — APPROVED** [review]

**Review by keeganquigley — APPROVED** [review]

**Review by dsm-w3f — APPROVED** [review]

**Comment by coreggon11** [comment]
Hello @Noc2 , I have merged main into this branch to resolve conflicts, could you let me know if everything is OK, or if you need some more information? Thank you

**Comment by Noc2** [comment]
@coreggon11 Thanks. We are currently waiting for one more approval from a committee member.

**Review by SBalaguer — APPROVED** [review]

**Pull request merged**
